components/productions/index/events.vue

```diff
@@ -55,6 +55,14 @@
           {{ $t('productions.overview.cancel') }}
         </b-button>
       </div>
+      <b-alert
+        v-for="(errorMessage, index) in errorMessages"
+        :key="index"
+        variant="danger"
+        :show="errorMessages.length > 0"
+        dismissible
+        >{{ errorMessage }}</b-alert
+      >
```

Comment:
Seems like a good candidate for a component in the ui library by the way?
If this PR is urgent it doesn't have to block it though.
Just wanted to point out that probably every usage of a `b-...` component should be wrapped at some point?

Reply:
done